src/pages/developing/keyboard.md

```diff
@@ -7,7 +7,55 @@ previousUrl: '/docs/developing/hardware-back-button'
 
-## inputmode
+## Working with Forms
+
+Keyboards and forms in single page applications present a unique challenge given how each platform tries to automatically resize webviews or scroll inputs into view. The biggest challenge is ensuring that an input is not covered by the keyboard when it is focused. Ionic Framework provides a few utilities that attempt to bridge the gap between the browser's default behavior and what is typically expected by users in mobile applications.
+
+### iOS WebView Resizing
+
+In Capacitor and Cordova applications, the default iOS webview behavior is to resize the webview when the on-screen keyboard is presented. When the keyboard is dismissed, the webview will then resize back to its full height. As a result, you may notice a flash of black or white where the keyboard used to be. This is empty space, and the webview will quickly resize to take up this space. 
+
+This flash may be undesirable to some developers. To avoid this, developers can change the resize mode for the Keyboard plugin in either Capacitor or Cordova. For example, developers may wish to disable all webview resizing by modifying their `capacitor.config.json` file:
+
+```json
+{
+  "plugins": {
+    "Keyboard": {
+      "resize": "none"
+    }
+  }
+}
+```
+
+> See [Capacitor Keyboard Configuration](https://capacitorjs.com/docs/apis/keyboard#keyboard-configuration-ios-only) for a full list of options. 
+
+There are still a few issues that exist when it comes to ensuring an input is not covered by the keyboard. This is where the [Scroll Assist](#scroll-assist) and [Scroll Padding](#scroll-padding) utilities can help.
+
+### Scroll Assist
+
+When an input is focused in web content on iOS, the browser will attempt to scroll the input into focus so it is not covered by the keyboard. Unfortunately, this scrolling is done on the `body` which results in your `ion-header` being pushed off screen. Ionic Framework suppresses this behavior and scrolls the input into focus by scrolling on `ion-content` through the Scroll Assist utility.
+
+The general flow of Scroll Assist is to determine how much we need to scroll by, suppress the browser's default scrolling behavior, and then scroll the input into view. However, there are a few points to be aware of. These are not bugs, but rather limitations in handling inputs in webviews:
+
+1. When running your app in a browser or as a PWA on older versions of iOS, the [Keyboard Lifecycle Events](#keyboard-lifecycle-events) may not be supported. As a result, the amount Scroll Assist scrolls by may be more than necessary as it can only estimate the keyboard height. Additionally, there may be a delay before Scroll Assist scrolls the input into view as it does not know exactly when the keyboard opens. This does not impact Capacitor or Cordova applications.
```

Comment:
You mentioned we get issues opened about these scenarios, but I'm having trouble visualizing them. hate to ask, but would small GIFs for each be helpful? as a way for someone to say, "oh yeah, that's exactly what I'm seeing." then ideally they don't file a new issue, or close it out since there's not much to be done.

Reply:
Yeah it's a good question. On the one hand I don't want there to be a whole section of our docs dedicated to showing off jank, but also the general consensus around the issues I get is that people just feel that things are "off".

Reply:
I think the overall purpose of this document is a) highlight how these utilities work and what problems they solve and b) say that the utility is never going to be perfect, but I'm not sure how to clearly explain b).

Reply:
haha good call, that's the drawback... could make us look bad. ok nevermind :)

Reply:
I'm open to brainstorming other ideas though. The issue for me is that this utility has been undocumented for several years yet it's responsible for a really important part of ionic apps. So even if we just document the functionality and work on the quirks for a later time that's fine by me.